packages/tokens/RELEASE.md

```diff
@@ -10,7 +10,7 @@ The "bump" step increases the _SemVer_ version number in the `package.json` file
 
 * Make sure your local `main` branch is up to date.
 * Create new custom branch from `main`.
-* `cd /design-system-tokens`
+* `cd /packages/tokens`
 * Run `yarn bump` and choose the _SemVer_ version as agreed upon on the previous PR.
```

Comment:
I think there's more to do here, but I'm not 100% sure what given that the [release scripts]https://github.com/hashicorp/design-system/blob/11f1c898e780092e7ce08d2f8c41632b61f9d7c6/packages/tokens/package.json#L22-L27) have some steps in them that our other packages did not and I have not personally released this package before. Either way, the version bumping should happen as part of using changesets and not required the manual command here.

Reply:
Similar to https://github.com/hashicorp/design-system/pull/11 the ideal state is that these instructions aren't necessary, but not 100% due to the caveat mentioned in my previous comment.

Reply:
Gotcha. Also haven't tested this particular release to npm 

It could be useful to test all the packages/ bumps e2e (maybe a `x.x.x.beta.0`, and then immediately deprecate in npm), so that I can update the docs accurately. But vaguely remembered you did not want to do that. Thoughts?

Reply:
Ok, I'll try it again

Reply:
@Dhaulagiri @didoo Update: Verdaccio doesn't really allow me to test with Changesets like I meant, so it's a bit N/A. Whenever a bump comes, will be good to test. 

Took another pass at the docs 3d10cedfab032370441419aa51e5013407f8f385, but please let me know if I missed anything. Ready for another review

Reply:
looking at this more closely I think we might need to pair on this a bit. we were doing some manual things here to avoid having `dist/` be in the path of the package that gets published which I doesn't work the same with these changes (nor would the previous logic have worked with changesets I don't think)

Reply:
I'll open a PR against this branch with an idea of what I'm talking about that might illustrate things better

Reply:
@Dhaulagiri I don't quite follow, let me try https://github.com/hashicorp/design-system/pull/40 out. Can ping on Slack in a bit